src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -4,74 +4,85 @@
   public void shooterInit() {
     SmartDashboard.putNumber(SMART_DASHBOARD_VELOCITY, motorVelocity);
     SmartDashboard.putNumber(SMART_DASHBOARD_TARGET_VELOCITY, targetVelocity);
-    SmartDashboard.putString(SMART_DASHBOARD_POSITION, positionOfArm.toString());
+    SmartDashboard.putNumber(SMART_DASHBOARD_POSITION, positionOfArm);
+    SmartDashboard.putNumber(SMART_DASHBOARD_TARGET_POSITION, targetArmPosition);
+
     leftShootMotor.setInverted(true);
+
+    MagnetSensorConfigs wristPositionMagnetConfigs = new MagnetSensorConfigs();
+    wristPositionMagnetConfigs.AbsoluteSensorRange = AbsoluteSensorRangeValue.Unsigned_0To1;
+    wristPositionMagnetConfigs.SensorDirection = SensorDirectionValue.CounterClockwise_Positive;
+    wristPositionMagnetConfigs.MagnetOffset = 0;  // TODO Calibrate this with a command
+    wristPositionEncoder.getConfigurator().apply(wristPositionMagnetConfigs);
+
+    FeedbackConfigs pivotConfigs = new FeedbackConfigs();
+    pivotConfigs.FeedbackRemoteSensorID = wristPositionEncoder.getDeviceID();
+    pivotConfigs.FeedbackSensorSource = FeedbackSensorSourceValue.RemoteCANcoder;
+    pivotConfigs.RotorToSensorRatio = 45 / 8;
+    pivotMotor.getConfigurator().apply(pivotConfigs);
+
+    TalonFX.optimizeBusUtilizationForAll(pivotMotor, leftShootMotor, rightShootMotor);
     stopMotors();
   }
 
   public void optomizeCan() {
     TalonFX.optimizeBusUtilizationForAll(pivotMotor, leftShootMotor, rightShootMotor);
   }
 
-  public void startMotors(double speed, Position position) {
+  public void setAngle(double angle) {
+    motionMagicVoltage = new MotionMagicVoltage(angle / 360);
+    motionMagicVoltage = motionMagicVoltage.withOverrideBrakeDurNeutral(true);
+    pivotMotor.setControl(motionMagicVoltage);
+    targetArmPosition = angle;
+  }
+
+  public void startMotors(double speed) {
     targetVelocity = speed;
-    positionOfArm = position;
-    if (position == Position.AMP) {
-      // motionMagicVoltage = new MotionMagicVoltage(-0.05);  // TODO: Change to fit amp angle -- not controled yet
-    } else if (position == Position.SPEAKER) {
-      // motionMagicVoltage = new MotionMagicVoltage(-0.1);  // TODO: Change to fit speaker angle -- not controled yet
-    }
     rightShootMotor.set(targetVelocity);
     leftShootMotor.set(targetVelocity);
   }
 
   public boolean canShoot() {
-    return motorsAtShootingSpeed && positionOfArm != Position.DEFAULT;
+    return motorsAtShootingSpeed && Math.abs(pivotMotor.getClosedLoopError().getValue()) <= TunerConstants.pivotMotorAngleErrorThreashhold;
   }
 
   public void setDefault() {
-    if (positionOfArm == Position.AMP) {
-      // motionMagicVoltage = new MotionMagicVoltage(0.05);  // TODO: Change to fit amp angle -- not controled yet
-    } else if (positionOfArm == Position.SPEAKER) {
-      // motionMagicVoltage = new MotionMagicVoltage(0.1);  // TODO: Change to fit speaker angle -- not controled yet
-    }
-    positionOfArm = Position.DEFAULT;
+    setAngle(0);
```

Comment:
Our angle won't be exactly 0 degrees. There's also the question of whether our default means "resting down" or "resting up". I think resting down is ~30 degrees and is probably fine for driving around with, especially because we need it there to load a new note anyway. The name `setDefault` might need to be specific as to what we consider the default state to be, but that's just a little nitpicky.